setup.cfg

```diff
@@ -145,10 +145,13 @@ ignore_errors=True
 [mypy-pandas.tests.extension.json.test_json]
 ignore_errors=True
 
+[mypy-pandas.tests.indexes.datetimes.test_tools]
+ignore_errors=True
+
 [mypy-pandas.tests.indexes.test_base]
 ignore_errors=True
 
-[mypy-pandas.tests.indexing.test_loc]
+[mypy-pandas.tests.scalar.period.test_period]
```

Comment:
this is now giving `error: Module has no attribute "__version__"` following #29212

again could add `#type: ignore`, see https://github.com/pandas-dev/pandas/pull/28339/commits/cfc2ef78d24aab0fbc4efd838e702ef8cbecb19f